ui/src/message_popup/playtest/smithA_scenario.jsx

```diff
@@ -153,6 +153,10 @@ Okay! Ready to start?`
 <div>Tim: “I’m not following you at all. How’d you do that?”</div>
 <br />
 <div>George: “Here, watch.”</div>
+<br />
+<div>Mr. Smith: "George, why don't you try explaining what you're doing to Tim?"</div>
+<br />
+<div>George: "Sure Mr. Smith. Okay Tim, so to modify that part first I..."</div>
```

Comment:
This is minor, but I'd suggest including a whole sentence here.  Then the "next" button will take them away to the next scene and cut it off anyway.